src/components/ParticleIDInput.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import * as React from 'react';
+import { action, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import TextBlock from '../common/TextBlock';
+import Button from '../common/buttons/Button';
+import TextField from './TextField';
+import { TYPOGRAPHY } from '../theme';
+
+const styles = StyleSheet.create({
+  descriptionText: {
+    ...TYPOGRAPHY.paragraph,
+    textAlign: 'center',
+  },
+  expandText: {
+    ...TYPOGRAPHY.small,
+    textAlign: 'center',
+    textDecorationLine: 'underline',
+  },
+});
+
+type Props = {|
+  onChange: (value: string) => void,
+  onContinuePress: () => void,
+  value: string,
+|};
+
+@observer
+class ParticleIDInput extends React.Component<Props> {
+  @observable _isExpanded: boolean = false;
+
+  @action
+  _expand = () => {
+    this._isExpanded = true;
+  };
+
+  render() {
+    const { onChange, onContinuePress, value } = this.props;
+
+    return (
+      <View>
+        {!this._isExpanded && (
+          <TouchableOpacity onPress={this._expand}>
+            <Text style={styles.expandText}>
+              I know my internal Brewskey box ID
+            </Text>
+          </TouchableOpacity>
+        )}
+        {this._isExpanded && [
+          <TextBlock
+            key="descriptionText"
+            textStyle={styles.descriptionText}
+            paddedBottom
+          >
+            Enter the hardware ID of your Brewskey box. We&#8217;ll skip the
```

Comment:
I'd rather you write the text like this `{"Test test asdf's test."}`. It's a lot more readable than the html code.